test/services/differential_expression_service_test.rb

```diff
@@ -265,4 +266,16 @@ class DifferentialExpressionServiceTest < ActiveSupport::TestCase
                                     test_array: @@studies_to_clean)
     assert_not DifferentialExpressionService.study_eligible?(empty_study)
   end
+
+  test 'should not mark studies with author de as eligible' do
```

Comment:
For consistent visual parsing:

```suggestion
  test 'should not mark studies with author DE as eligible' do
```